syntax = "proto2";

package Metrics;

message Metric {
	required string name = 1;
	required double value = 2;
}

message Metrics { // per test
	repeated Metric measurements = 1;
}

message MetricsSet {
	repeated Metrics metrics = 1;
	required int32 tests = 2;
	required int32 solved = 3;
}